### PR TITLE
Readme: how to set up development environment

### PR DESCRIPTION
**Pull request by leovdd**
** Added more instructions on how to set up your development bot, get it accepted into a channel and create your .env file **

### CODE REVIEW

**Review by Eva774 — DISMISSED** [review]

**Review by yannicfreson — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -5,26 +5,62 @@ This is a Python-based discord bot developed by the nerdland fan community.
...
 To get this project up and running, make sure you have the following installed:
 - Python 3
 - PIP
 - [Poetry](https://python-poetry.org/docs/#installation)
+- Use your favourite ide and git to clone the nerdlandbot to your local machine
 
-Once you have these installed (you can check by running 'python --version', 'pip -V' and 'poetry -V' in a commandline) run the following command to install the required packages:
+Once you have these installed (you can check by running 'python --version', 'pip -V' and 'poetry -V' in a commandline), add the required packages (see requirements.txt) using pip:
+```
+pip install <package name>
```
Comment (Autom3) [inline README.md]: Can be done in one command using `pip install -r requirements.txt`

File: README.md [hunk 2]
```diff
@@ -5,26 +5,62 @@ This is a Python-based discord bot developed by the nerdland fan community.
...
-# Getting started
+# Setting up your development environment
 To get this project up and running, make sure you have the following installed:
 - Python 3
 - PIP
 - [Poetry](https://python-poetry.org/docs/#installation)
+- Use your favourite ide and git to clone the nerdlandbot to your local machine
 
-Once you have these installed (you can check by running 'python --version', 'pip -V' and 'poetry -V' in a commandline) run the following command to install the required packages:
+Once you have these installed (you can check by running 'python --version', 'pip -V' and 'poetry -V' in a commandline), add the required packages (see requirements.txt) using pip:
```
Comment (Autom3) [inline README.md]: Maybe clarify they can use either pip or poetry but shouldn't try to use both

Comment (leovdd) [reply README.md]: I didn't try doing pip install only myself. When running poetry afterwards, I noticed a lot of other things were still being installed. Are you sure pip install -r requirements will  do everything required ?

File: README.md (hunk 1 above)
Comment (leovdd) [reply README.md]: Ok changed and committed.

File: README.md (hunk 2 above)
Comment (Autom3) [reply README.md]: Pip will have installed everything globally, whereas poetry installs everything in a virtual environment, so you will indeed have installed everything twice

**Comment by woutermeuwis** [comment]
Looks good, but Tom's latest comment has not yet been implemented

**Comment by leovdd** [comment]
I have adjusted. 
Hooh sensei, please to approve or comment.

**Review by bobvdvleuten — APPROVED** [review]

**Review by Autom3 — APPROVED** [review]

**Review by woutermeuwis — APPROVED** [review]

**Pull request merged**